resource-validator/src/main/scala/com/broadinstitute/dsp/resourceValidator/ErroredRuntimeChecker.scala

```diff
@@ -32,8 +32,9 @@ object ErroredRuntimeChecker {
 
       def checkDataprocCluster(runtime: Runtime, isDryRun: Boolean)(implicit ev: Ask[F, TraceId]): F[Option[Runtime]] =
         for {
-          clusterOpt <- deps.dataprocService
+          clusterOpt <- deps.runtimeCheckerDeps.dataprocService
             .getCluster(runtime.googleProject, regionName, DataprocClusterName(runtime.runtimeName))
+          isBillingEnabled <- deps.billingService.isBillingEnabled(runtime.googleProject)
```

Comment:
thanks for addressing ErroredRuntime as well!

Same comment for moving isBillingEnabled check down